Project/ICSharpCode.TextEditor.csproj

```diff
@@ -264,12 +150,14 @@
     <EmbeddedResource Include="Resources\Lean.xshd" />
     <EmbeddedResource Include="Resources\Lisp.xshd" />
     <EmbeddedResource Include="Resources\Lua.xshd" />
+    <EmbeddedResource Include="Resources\Mode.xsd" />
```

Comment:
This file has a different extension. Is that intentional?

Reply:
💭 It matches the remove line above.

Reply:
Yes, this is the schema file